Maths/Sigmoid.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @see [Wikipedia](https://en.wikipedia.org/wiki/Sigmoid_function)
+ * Calculate the Sigmoid value of a given number.
+ * @param {number} x- The input number.
+ * @returns {number} The Sigmoid value of the input.
+ */
+
+const sigmoid = (x) => {
```

Comment:
if you omit the `{}`, you can shorten this to `const sigmoid = x => 1 / (1 + Math.exp(-x))`

Reply:
Hello, 
I have done the required reformatting of the function.

Thank You